pkg/distributor/otel.go

```diff
@@ -429,7 +429,7 @@ func otelMetricsToTimeseries(ctx context.Context, tenantID string, addSuffixes b
 // Old, less efficient, version of otelMetricsToTimeseries.
 func otelMetricsToTimeseriesOld(ctx context.Context, tenantID string, addSuffixes bool, discardedDueToOtelParseError *prometheus.CounterVec, logger log.Logger, md pmetric.Metrics) ([]mimirpb.PreallocTimeseries, error) {
 	converter := prometheusremotewrite.NewPrometheusConverter()
-	errs := converter.FromMetrics(ctx, md, prometheusremotewrite.Settings{
+	_, errs := converter.FromMetrics(ctx, md, prometheusremotewrite.Settings{
```

Comment:
note to reviewers: I'm not sure what we'd do with annotations here @aknuds1

Reply:
You can do this:
```golang
	annots, errs := converter.FromMetrics(ctx, md, prometheusremotewrite.Settings{
		AddMetricSuffixes:                   addSuffixes,
		EnableCreatedTimestampZeroIngestion: enableCTZeroIngestion,
	})
	promTS := converter.TimeSeries()
	if errs != nil {
                ...
		level.Warn(logger).Log("msg", "OTLP parse error", "err", parseErrs)
	}
	ws, _ := annots.AsStrings("", 0, 0)
	if len(ws) > 0 {
		level.Warn(logger).Log("msg", "Warnings translating OTLP metrics to Prometheus write request", "warnings", ws)
	}
```

Reply:
followup PR? there's already a lot going on in this PR